packages/core/src/execution-engine/node-execution-context/execute-context.ts

```diff
@@ -183,7 +184,10 @@ export class ExecuteContext extends BaseExecuteContext implements IExecuteFuncti
 
 	logNodeOutput(...args: unknown[]): void {
 		if (this.mode === 'manual') {
-			this.sendMessageToUI(...args);
+			const parsedLogArgs = args.map((arg) =>
+				typeof arg === 'string' ? jsonParse(arg, { fallbackValue: true }) : arg,
```

Comment:
I think the fallback value "true" is coming from here when logging strings, see comment https://github.com/n8n-io/n8n/pull/15111#issuecomment-2851135734

Reply:
Fat-fingered the arg :facepalm:

Added a test, thanks for catching this!